bin/pycbc_multi_inspiral

```diff
@@ -86,89 +110,254 @@ psd.insert_psd_option_group_multi_ifo(parser)
 scheme.insert_processing_option_group(parser)
 fft.insert_fft_option_group(parser)
 
+#Adding these to see if it lets me use the gating options
+from pycbc.vetoes.sgchisq import SingleDetSGChisq
+pycbc.opt.insert_optimization_option_group(parser)
+pycbc.weave.insert_weave_option_group(parser)
+pycbc.inject.insert_injfilterrejector_option_group(parser)
+SingleDetSGChisq.insert_option_group(parser)
 opt = parser.parse_args()
 
+generate_histograms = False
+generate_snr_plots = False
+generate_chi2_plots = False
+# Polarisation doesn't matter for coherent matched filter so set to zero.
+pol = 0
+# Use the time in the middle of the segment to calculate the antenna patterns.
+t_gps = (opt.trig_end_time[0] + opt.trig_start_time[0])/2
+
+# Set sky position being searched.
+ra = opt.longitude 
+dec = opt.latitude
+
+# Put the ifos in alphabetical order so they are always called in
+# the same order.
+opt.instruments.sort()
+
 strain.verify_strain_options_multi_ifo(opt, parser, opt.instruments)
 strain.StrainSegments.verify_segment_options_multi_ifo(opt, parser,
                                                        opt.instruments)
 psd.verify_psd_options_multi_ifo(opt, parser, opt.instruments)
 scheme.verify_processing_options(opt, parser)
 fft.verify_fft_options(opt,parser)
 
-if opt.verbose:
-    log_level = logging.DEBUG
-else:
-    log_level = logging.WARN
+log_level = logging.DEBUG if opt.verbose else logging.INFO
 logging.basicConfig(format='%(asctime)s : %(message)s', level=log_level)
 
 ctx = scheme.from_cli(opt)
 
+def get_weighted_antenna_patterns(Fp_dict,Fc_dict,sigma_dict):
+    """
+    Output: wp: 1 x nifo of the weighted antenna response fuctions to plus 
+                polarisation for each ifo
+            wc: 1 x nifo of the weighted antenna response fuctions to cross
+                polarisation for each ifo
+    Input:  Fp_dict: Dictionary of the antenna response fuctions to plus
+                     polarisation for each ifo
+            Fc_dict: Dictionary of the antenna response fuctions to cross
+                     polarisation for each ifo
+           sigma_dict: Sigma dictionary for each ifo (sensitivity of each ifo)
+    """
+    #Need the keys to be in alphabetical order
+    keys = sorted(sigma_dict.keys())
+    wp = np.array([sigma_dict[ifo]*Fp_dict[ifo] for ifo in keys])
+    wc = np.array([sigma_dict[ifo]*Fc_dict[ifo] for ifo in keys])
+    return wp,wc
+
+def get_projection_matrix(wp,wc):
+    """
```

Comment:
*This* function is why you don't need to worry about polarization ... (Not a review request, just a comment).